packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -1818,6 +1813,8 @@ const VideoPlayerBase = class extends React.Component {
 			proportionSelection = selection.map(t => t / this.state.duration);
 		}
 
+		const durFmt = this.getDurFmt(locale);
```

Comment:
Pushed a change to remove setting `durFmt` to an instance member in order to keep our render methods idempotent. Since the method is memoized, the cost of invoking it elsewhere is only the lookup cost.